### ISSUE TITLE
could you please tell me about the partition proportion of training, validation, and testing dataset.

### ISSUE THREAD

**Issue by chriszxk** [VincentHancoder/SSGD#3]
Hi:

could you please tell me about the partition proportion of training, validation, and testing dataset.

Thanks,
chris

**Comment by VincentHancoder** [comment]
Hello, the proportion we use to divide dataset is in the ratio of 4:1:1, we use the validation dataset as testing dataset as well. Meanwhile, you can check the same result of train files and corresponding validation files in annotation folder(e.g. train1.json and val1.json).

**Comment by chriszxk** [comment]
> Hello, the proportion we use to divide dataset is in the ratio of 4:1:1, we use the validation dataset as testing dataset as well. Meanwhile, you can check the same result of train files and corresponding validation files in annotation folder(e.g. train1.json and val1.json).

So,  is the accuracy (AP.5, AP75) computed on validation dataset in your paper,

**Comment by VincentHancoder** [comment]
Yes, and we use the 5-fold cross-validation method to obtain and calculate AP values.

**Comment by chriszxk** [comment]
> Yes, and we use the 5-fold cross-validation method to obtain and calculate AP values.

Hello, I noticed that there is a class called "OK" in the dataset. I would like to know if you included the "OK" objects in the training process.

**Comment by hs19991218** [comment]
请问在训练当中是如何解决7种缺陷数量不均衡的问题呢？

**Comment by FisherYuuri** [comment]
有一些类别前后不一致，数量也不一致

**Comment by WBFXX** [comment]
> > Yes, and we use the 5-fold cross-validation method to obtain and calculate AP values.
> 
> Hello, I noticed that there is a class called "OK" in the dataset. I would like to know if you included the "OK" objects in the training process.

Did you find the answer to this question?

**Comment by zrainyyy** [comment]
> > > Yes, and we use the 5-fold cross-validation method to obtain and calculate AP values.
> > 
> > 
> > Hello, I noticed that there is a class called "OK" in the dataset. I would like to know if you included the "OK" objects in the training process.
> 
> Did you find the answer to this question?

 Did you find the answer to this question?
